etc/linters.json

```diff
@@ -2,27 +2,158 @@
     },
     {
         "name": "markdown-link-check",
         "args": "--config /etc/opt/goat/links.json --verbose",
-        "filetype": ".md",
+        "filetype": [
+            ".md"
+        ],
         "installer": "npm",
-        "executor": "npx"
+        "executor": "npx",
+        "env": "MARKDOWN_LINK_CHECK_CONFIG"
     },
     {
         "name": "cspell",
         "args": "-c /etc/opt/goat/cspell.config.js -v --no-must-find-files --",
-        "filetype": ".md",
+        "filetype": [
+            ".md"
+        ],
+        "installer": "npm",
+        "executor": "npx",
+        "env": "CSPELL_CONFIG"
+    },
+    {
+        "name": "cfn-lint",
+        "args": "-D --config-file /etc/opt/goat/.cfnlintrc.yml",
+        "filetype": [
+            ".yml",
+            ".yaml"
+        ],
+        "installer": "pip",
+        "executor": "pipenv run",
+        "env": "CFN_LINT_CONFIG"
+    },
+    {
+        "name": "jscpd",
+        "args": "--config /etc/opt/goat/.jscpd.json *",
+        "filetype": [
+            "all"
+        ],
+        "installer": "npm",
+        "executor": "npx",
+        "env": "JSCPD_CONFIG"
+    },
+    {
+        "name": "actionlint",
+        "args": "",
+        "filetype": [
+            ".yml",
+            ".yaml"
+        ],
+        "installer": "docker",
+        "env": "ACTIONLINT_CONFIG"
+    },
+    {
+        "name": "markdownlint",
+        "args": "-c /etc/opt/goat/.markdown-lint.yml",
+        "filetype": [
+            ".md"
+        ],
+        "installer": "npm",
+        "env": "MARKDOWNLINT_CONFIG"
+    },
+    {
+        "name": "textlint",
+        "args": "-c /etc/opt/goat/.textlintrc \"/goat/**\"",
+        "filetype": [
+            "all"
+        ],
         "installer": "npm",
-        "executor": "npx"
+        "executor": "npx",
+        "env": "TEXTLINT_CONFIG"
+    },
+    {
+        "name": "black",
+        "args": "--config /etc/opt/goat/.python-black --diff --check",
+        "filetype": [
+            ".py"
+        ],
+        "installer": "pip",
+        "executor": "pipenv run",
+        "env": "BLACK_CONFIG"
+    },
+    {
+        "name": "mypy",
+        "args": "--config-file /etc/opt/goat/.mypy.ini --install-types --non-interactive",
+        "filetype": [
+            ".py"
+        ],
+        "installer": "pip",
+        "executor": "pipenv run",
+        "env": "MYPY_CONFIG"
+    },
+    {
+        "name": "shfmt",
+        "args": "-d",
+        "filetype": [
+            ".sh"
+        ],
+        "installer": "docker",
+        "env": "SHFMT_CONFIG"
+    },
+    {
+        "name": "shellcheck",
+        "args": "-C -x",
+        "filetype": [
+            ".sh"
+        ],
+        "installer": "docker",
+        "env": "SHELLCHECK_CONFIG"
+    },
+    {
+        "name": "yamllint",
+        "args": "-c /etc/opt/goat/.yamllint.yml",
+        "filetype": [
+            ".yaml",
+            ".yml"
+        ],
+        "installer": "pip",
```

Comment:
Isn't the installer technically `pipenv`, i.e. `pipenv install`?  Does it really matter, though?  Same for the others here; I don't know where the installer matters.

Reply:
I'm going to remove the installer member. I was only using that to handle dynamic installation in the Dockerfile. Since we're not doing that, I don't see the need, either.